admin/app/components/users/user-detail-personal-information/authentication-method.hbs

```diff
@@ -1,95 +1,122 @@
-      />
-    {{/if}}
-  </div>
-  {{#if (and @user.hasUsernameAuthenticationMethod (not @user.hasOnlyOneAuthenticationMethod))}}
-    <PixButton
-      class="user-authentication-method__remove-button"
-      @triggerAction={{fn @toggleDisplayRemoveAuthenticationMethodModal "USERNAME"}}
-      data-test-remove-username
-    >
-      SUPPRIMER
-    </PixButton>
-  {{/if}}
-</div>
-<div class="user-authentication-method__item" data-test-pole-emploi>
-  <div class="user-authentication-method-item__label">
-    <span>Pôle Emploi</span>
-    {{#if @user.hasPoleEmploiAuthenticationMethod}}
-      <FaIcon @icon="check-circle" class="user-authentication-method-item__check" />
-    {{else}}
-      <FaIcon
-        @icon="times-circle"
-        aria-label="L'utilisateur n'a pas de méthode de connexion avec Pôle Emploi"
-        class="user-authentication-method-item__uncheck"
-      />
-    {{/if}}
-  </div>
-  {{#if (and @user.hasPoleEmploiAuthenticationMethod (not @user.hasOnlyOneAuthenticationMethod))}}
-    <PixButton
-      class="user-authentication-method__remove-button"
-      @triggerAction={{fn @toggleDisplayRemoveAuthenticationMethodModal "POLE_EMPLOI"}}
-      data-test-remove-pole-emploi
-    >
-      SUPPRIMER
-    </PixButton>
-  {{/if}}
-</div>
-<div class="user-authentication-method__item" data-test-mediacentre>
-  <div class="user-authentication-method-item__label">
-    <span>Médiacentre</span>
-    {{#if @user.hasGARAuthenticationMethod}}
-      <FaIcon @icon="check-circle" class="user-authentication-method-item__check" />
-    {{else}}
-      <FaIcon
-        @icon="times-circle"
-        aria-label="L'utilisateur n'a pas de méthode de connexion avec GAR"
-        class="user-authentication-method-item__uncheck"
-      />
-    {{/if}}
-  </div>
-  {{#if (and @user.hasGARAuthenticationMethod (not @user.hasOnlyOneAuthenticationMethod))}}
-    <PixButton
-      class="user-authentication-method__remove-button"
-      @triggerAction={{fn @toggleDisplayRemoveAuthenticationMethodModal "GAR"}}
-      data-test-remove-mediacentre
-    >
-      SUPPRIMER
-    </PixButton>
-  {{/if}}
-</div>
\ No newline at end of file
+
+<table>
+
+  <caption class="authentication-method__caption">Méthodes de connexion</caption>
+
+  <tbody>
+    <tr data-test-email>
+      <td>Adresse e-mail</td>
+      <td>
+        {{#if @user.hasEmailAuthenticationMethod}}
+          <FaIcon
+            @icon="check-circle"
+            aria-label="L'utilisateur a une méthode de connexion avec adresse e-mail"
+            class="authentication-method__check"
+          />
+        {{else}}
+          <FaIcon
+            @icon="times-circle"
+            aria-label="L'utilisateur n'a pas de méthode de connexion avec adresse e-mail"
+            class="authentication-method__uncheck"
+          />
+        {{/if}}
+      </td>
+      <td>
+        {{#if (and @user.hasEmailAuthenticationMethod (not @user.hasOnlyOneAuthenticationMethod))}}
```

Comment:
Est-ce qu'on sortirait pas ces conditions en une seule condition ? Ca fait de la logique métier dans le composant et de ce que je comprends ce serait quelquechose comme `userHasRightToRemoveEmailAuthenticationMethod`
`(and @user.hasEmailAuthenticationMethod (not @user.hasOnlyOneAuthenticationMethod))`

Reply:
c'est ajouté

Reply:
`userHasRightToRemoveEmailAuthenticationMethod`, je suis pas convaincue de la partie `userHasRight`
`allowToRemoveEmailAuthenticationMethod` ? Peut être pas mieux, d'autres idées ?